go/common/grpc/policy/policy_test.go

```diff
@@ -151,20 +160,27 @@ func TestAccessPolicy(t *testing.T) {
 	serverCertPool.AddCert(serverX509Cert)
 
 	// Create a new gRPC server.
-	serverConfig := &ServerConfig{
+	serverConfig := &cmnGrpc.ServerConfig{
 		Name:          host,
 		Port:          port,
 		Certificate:   serverTLSCert,
-		CustomOptions: []grpc.ServerOption{grpc.CustomCodec(&CBORCodec{})},
+		CustomOptions: []grpc.ServerOption{grpc.CustomCodec(&cmnGrpc.CBORCodec{})},
 	}
-	grpcServer, err := NewServer(serverConfig)
+	grpcServer, err := cmnGrpc.NewServer(serverConfig)
 	require.NoErrorf(err, "Failed to create a new gRPC server: %v", err)
 
 	// Create a new pingServer with a new RuntimePolicyChecker.
-	policyChecker := NewDynamicRuntimePolicyChecker()
+	watcher := NewPolicyWatcher()
+	// Register the PolicyWatcherService.
+	api.RegisterService(grpcServer.Server(), watcher)
+
+	policyChecker := NewDynamicRuntimePolicyChecker("ping", watcher)
```

Comment:
Shouldn't `"ping"` be `"PingService"`? Maybe change it to use `serviceDesc.ServiceName`?